tests/foreman/api/test_errata.py

```diff
@@ -184,10 +184,7 @@ def test_positive_install_multiple_in_host(
 
     :CaseLevel: System
     """
-    ak = target_sat.api.ActivationKey(
-        organization=function_org,
-        environment=function_lce,
-    ).create()
+    ak = target_sat.api.ActivationKey(organization=function_org).create()
```

Comment:
AK will require either function_lce or cv for contents, here only register doesn't requires lce when there's a AK

Reply:
I received an error at registration without passing lce-
`HTTP error (500 - Internal Server Error): At least one activation key must have a lifecycle environment and content view assigned to it\n') `

Reply:
yes, that must be due to this change, either lce or cv should be assigned to AK